src/Calculator/Views/MainPage.xaml.cs

```diff
@@ -61,24 +62,37 @@ public MainPage()
 
             UserActivityRequestManager.GetForCurrentView().UserActivityRequested += async (_, args) =>
             {
-                var deferral = args.GetDeferral();
-                if (deferral == null)
+                using (var deferral = args.GetDeferral())
                 {
-                    // Windows Bug in ni_moment won't return the deferral propoerly, see https://microsoft.visualstudio.com/DefaultCollection/OS/_workitems/edit/47775705/
-                    return;
+                    if (deferral == null)
+                    {
+                        // Windows Bug in ni_moment won't return the deferral propoerly, see https://microsoft.visualstudio.com/DefaultCollection/OS/_workitems/edit/47775705/
```

Comment:
Nit: It seems the bug has been resolved. Maybe we can consider removing this later.

Reply:
Let's keep it at here, since it was an OS bug. Added a trace to track the actual behavior.